src/scss/_mixins.scss

```diff
@@ -91,51 +91,74 @@
 			top: $_o-layout-gutter;
 		}
 	}
+
+	.o-layout__navigation li {
+		list-style: none;
+	}
+
 	.o-layout__navigation ul,
 	.o-layout__navigation ol {
-		border: 0;
-		border-left: 2px solid oColorsGetPaletteColor('teal');
 		padding: 0;
+	}
+
+	.o-layout__navigation > ul,
+	.o-layout__navigation > ol {
+		border: 0;
 		text-align: left;
-		list-style: none;
 		margin: 0 0 oTypographySpacingSize(7);
 
 		@include oGridRespondTo($from: M) {
-			border-left: 0;
 			text-align: right;
 			margin-top: $_o-layout-gutter;
 		}
 
-
 		li {
-			&.o-layout__navigation-title {
-				@include oTypographyBold('sans');
+			&:not(.o-layout__navigation-title) [aria-current="location"] {
+				color: oColorsGetPaletteColor('teal-30');
+				background: oColorsMix('teal', 'white', 20);
 			}
 
-			&:not(.o-layout__navigation-title) {
-				[aria-current="location"] {
-					color: oColorsGetPaletteColor('teal-30');
-					background: oColorsMix('teal', 'white', 20);
+			a {
+				@include oTypographySans(1);
+				@include oTypographyLink();
+				border: inherit;
+				display: inline-block;
+				padding: 0.4em 1em;
+				word-wrap: anywhere;
```

Comment:
what is _anywhere_ ?!

Reply:
>To prevent overflow, an otherwise unbreakable string of characters — like a long word or URL — may be broken at any point if there are no otherwise-acceptable break points in the line. No hyphenation character is inserted at the break point. Soft wrap opportunities introduced by the word break are considered when calculating min-content intrinsic sizes.
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

So I guess similar to `break-word`, but more squishy.